garage/tf/models/autopickable.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
Should the file rename to `auto_pickable.py`?

Reply:
yes that sounds better.

Reply:
generally underscores follow whatever CamelCase you use in the class name

`Autopickleable` --> `autopickleable.py`
`AutoPickleable` --> `auto_pickleable.py`